exceptions/exceptions.cc

```diff
@@ -70,6 +70,10 @@ read_write_timeout_exception::read_write_timeout_exception(exception_code code,
     , block_for{block_for}
     { }
 
+view_update_generation_timeout_exception::view_update_generation_timeout_exception()
+    : request_timeout_exception{
+          exception_code::WRITE_TIMEOUT, "Request timed out - couldn't generate materialized view updates in time"} {}
```

Comment:
Here and in the comment for this class below, it is confusing what "generating" view updates mean. Does it include waiting for the remote view replica to answer? It should be explained.

Reply:
I changed the comment, and the error message is now:
```
Request timed out - couldn't prepare materialized view updates in time
```
`prepare` signifies that it isn't about sending, the timeout happened earlier, at the "preparation" stage

Reply:
But why would the preparation phase take a long time? Were we waiting for the read-before-write?